src/auth/index.js

```diff
@@ -3,18 +3,21 @@
 import { createStorage } from './storage';
 import { createProviders } from './providers';
 import { StitchError } from '../errors';
+import * as authCommon from './common';
 import * as common from '../common';
 
 const jwtDecode = require('jwt-decode');
 
 export default class Auth {
   constructor(client, rootUrl, options) {
```

Comment:
could just use default values in constructor instead of using the `Object.assign` below. You can nest default values. 

`constructor(client, rootUrl, options = {storageType = 'localStorage', codec = authCommon.APP_CLIENT_CODEC})`

this gives a good idea to the consumer what kind of options this can take.

Reply:
This won't work when only one option is specified. In this case the codec would get unset.